mod_reforged/hooks/msu.nut

```diff
@@ -320,4 +320,16 @@
 	}
 });
 
+::logInfo("Reforged::MSU -- adding ::MSU.Array.removeValues");
+::MSU.Array.removeValues <- function( _array, _values )
```

Comment:
This can be done natively in squirrel via:
```squirrel
myArray = myArray.filter(@(_, a) excludeArray.find(a) != null);
```
However one can argue that the gain of this function is more convenience and/or streamlined code as one can do:
```squirrel
::MSU.Array.removeValues(myArray, [1, 2, 3, 4]);

// instead of

local exclude = [1, 2, 3, 4];
myArray = myArray.filter(@(_, a) exclude.find(a) != null);
```
I'd ask @Enduriel's opinion on this.

Reply:
We discussed this kind of thing internally in the MSU team and we seem to have settled on encouraging stuff like `filter` rather than adding minor functions for things like this, even though that was not the case historically.

Reply:
Is it ok to add the helper function locally just within the `item_names.nut`?
At the end of the day it still saves 1 line of code whenever it's used

Reply:
Actually now that I think about it I was wrong and these are not equivalent and adding this function makes sense, your function modifies the array in place which may sometimes be relevant. To me this makes sense and should stay as it is.